README.md

```diff
@@ -124,6 +125,92 @@ If there are no releases for more than 30 days and in `main` present something t
 * [:rebaseStalePrs](https://docs.renovatebot.com/presets-default/#rebasestaleprs) - Rebase existing PRs any time the base branch has been updated.
 * [Update package/GHA references in Markdown files](https://github.com/renovatebot/.github/blob/d9b3c1914f4bf9dbecc6456610ca89530260572f/default.json#L121-L140)
 
+### Custom check creation
+
+* [How to create a custom regex manager](https://www.cloudquery.io/how-to-guides/update-plugins-using-renovate)
+
+
+1. Create a regex manager in your `renovate.json5` like
+
+    ```json5
+    {
+      regexManagers: [
+        {
+          description: "",
+          fileMatch: [""], // regex
+          matchStrings: [
+            "", // Your regex will be here
+          ],
+          //One of https://docs.renovatebot.com/modules/datasource/
+          datasourceTemplate: "",
+          packageNameTemplate: "{{packageName}}",
+          // custom-prefix@ needed only in case you;d like provide some custom rules to theses packages
+          depNameTemplate: "custom-prefix@{{packageName}}",
+           // In case you'd like remove `v` prefix
+          extractVersionTemplate: "^v?(?<version>.*)$",
+          // One of https://docs.renovatebot.com/modules/versioning/, IE `semver`:
+          versioningTemplate: "semver",
+        },
+      ],
+      //// By providing `packageRules` and `matchDepPatterns` you can customize your package updates from regexManager.
+      // packageRules: [
+      //   {
+      //     matchDepPatterns: ["^custom-prefix@"],
+      //   },
+      // ],
+    }
+    ```
+
+    and fill all fields except `matchStrings`.
+
+2. Find file with dependencies that you want to update and past file content to [regex101.com](https://regex101.com)
+
+3. Write regex for `matchStrings` which will catch:
+
+    > **Note** Renovate uses RE2 without back and forward lookup groups support, so PCRE2 with these limitations in mind works fine
+
+    * current package version (`<currentValue>`)
+    * package source URL (`<packageName>`). What should be inside depends on `datasourceTemplate` which you will use.  
+      For example, for `datasourceTemplate: "github-releases"` you need cutch to `<packageName>` `GithubPkgOrg/PkgRepo` string.
+
+    <details><summary>Catch fields example</summary>
+
+    ![Regex catch example](docs/regex-example.png)
+
+    </details>
+
+4. Copy regex from `regex101.com` to `matchStrings` section. Replace all `\` by `\\`.
+
+5. Test that all works as expected. It could be done by creating temporary repo with your Renovate config and test file, with a non-latest version. Read Renovate logs, force PR creation, and check opened PR if needed.
```

Comment:
```suggestion
5. Test that all works as expected. It could be done by creating a temporary repo with your Renovate config and test file, with a non-latest version. Read Renovate logs, force PR creation, and check opened PR if needed.
```